src/Build/BackEnd/Components/Scheduler/Scheduler.cs

```diff
@@ -2434,7 +2434,7 @@ private void WriteNodeUtilizationGraphLine(ILoggingService loggingService, Build
 
             bool haveNonIdleNode = false;
             StringBuilder stringBuilder = new StringBuilder(64);
-            stringBuilder.AppendFormat("{0}:   ", previousEventTime.Ticks);
+            stringBuilder.Append(previousEventTime.Ticks).Append(":   ");
```

Comment:
This one surprises me a bit, what should I learn from this being preferred?

Reply:
Two things:
1. AppendFormat needs to parse the format string
2. AppendFormat takes object arguments, so Ticks is getting boxed into an object
3. On netfx, that boxed object will then have ToString called on it (on core it can do better).

The latter doesn't need to do any parsing, and Append has overloads for all the primitives, so it doesn't need to box. And on core, the formatting of the primitive is then done directly into the buffer.

If this were only targeting core, it could have also been:
```C#
stringBuilder.AppendFormat($"{previousEventTime}:    ");
```
which will take advantage of the interpolated string handler support in a dedicated AppendFormat overload.
https://devblogs.microsoft.com/dotnet/string-interpolation-in-c-10-and-net-6/